File: python_stack/Algos/Fundamentals/reverseString.py
def reverseString(str):
    s=list(str)
    start=0
    end=len(str)-1
    while(end>start):
        s[start],s[end]=s[end],s[start]
        start+=1
        end-=1
    return "".join(s)
print(reverseString("hello"))

File: python_stack/Python_OOP/Recursion/ReverseString.js
/**
 * @param {character[]} s
 * @return {void} Do not return anything, modify s in-place instead.
 */
function reverseString(str){

if (str.length<=0) return
console.log(str[str.length-1])
reverseString(str.substring(0,str.length-1))

}

reverseString("hello")